change/@fluentui-react-f7506cff-14ff-4780-9ee1-dd34032a5124.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## 🕵🏾‍♀️ [<b data-uid='vrt-gist'></b> visual regressions to review in the fluentuiv8 Visual Regression Report](https://aka.ms/visual-changes-review?organization=uifabric&project=fabricpublic&prId=33206&commitId=a3ab2317579084b2f54ab283e48b513a84deca78&env=prod&clientType=FLUENTUI)


  <details><summary><b>Button (compat)</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button (compat).Root RTL.chromium.png|145| Changed|
|Button (compat).Icon Only.chromium.png|115| Changed|
|Button (compat).No Icon.chromium.png|120| Changed|
|Button (compat).Checked.chromium.png|145| Changed|
|Button (compat).Root.chromium.png|145| Changed|

</details>


  <details><summary><b>Button Compound (compat)</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Compound (compat).Checked.chromium.png|376| Changed|
|Button Compound (compat).Root.chromium.png|376| Changed|
|Button Compound (compat).No Icon.chromium.png|376| Changed|
|Button Compound (compat).Root RTL.chromium.png|376| Changed|

</details>


  <details><summary><b>Button Special Scenarios (compat)</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Special Scenarios (compat).no flex shrink.chromium.png|699| Changed|

</details>


  <details><summary><b>Button Split (compat)</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Button Split (compat).Root.chromium.png|185| Changed|
|Button Split (compat).Root RTL.chromium.png|184| Changed|
|Button Split (compat).Checked.chromium.png|185| Changed|

</details>


  <details><summary><b>Callout</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Callout.Root.chromium.png|2309| Changed|
|Callout.No callout width specified.chromium.png|2319| Changed|
|Callout.No beak.chromium.png|2306| Changed|
|Callout.Top center.chromium.png|2178| Changed|
|Callout.Top left edge.chromium.png|2307| Changed|

</details>


  <details><summary><b>Coachmark</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Coachmark.Collapsed.chromium.png|137| Changed|

</details>


  <details><summary><b>ContextualMenu</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ContextualMenu.With submenus with hrefs.chromium.png|268| Changed|

</details>


  <details><summary><b>Dialog</b> 5 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Dialog.Blocking.chromium.png|120| Changed|
|Dialog.Root RTL.chromium.png|120| Changed|
|Dialog.Wide Dialog.chromium.png|120| Changed|
|Dialog.Large header.chromium.png|120| Changed|
|Dialog.Root.chromium.png|120| Changed|

</details>


  <details><summary><b>Image</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Image.No fit, only height.chromium.png|882| Changed|
|Image.No fit, no w-h.chromium.png|10759| Changed|
|Image.No fit, only width.chromium.png|31612| Changed|
|Image.Fit- none, image larger.chromium.png|10981| Changed|

</details>


  <details><summary><b>MessageBar</b> 8 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MessageBar.Root Actions RTL.chromium.png|231| Changed|
|MessageBar.Root Dismiss And Action RTL.chromium.png|231| Changed|
|MessageBar.Root Actions Single Line RTL.chromium.png|231| Changed|
|MessageBar.Root Dismiss And Action Single Line RTL.chromium.png|231| Changed|
|MessageBar.Root actions single line.chromium.png|231| Changed|
|MessageBar.Root dismiss and action.chromium.png|231| Changed|
|MessageBar.Root actions.chromium.png|231| Changed|
|MessageBar.Root dismiss and action single line.chromium.png|231| Changed|

</details>


  <details><summary><b>Panel</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Panel.With footer at the bottom.chromium.png|120| Changed|
|Panel.With custom navigation.chromium.png|126| Changed|

</details>


  <details><summary><b>ResizeGroup</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ResizeGroup.Root RTL.chromium.png|971| Changed|
|ResizeGroup.Root.chromium.png|971| Changed|

</details>


  <details><summary><b>react-charting-AreaChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-AreaChart.Custom Accessibility.chromium.png|11| Changed|

</details>


  <details><summary><b>react-charting-LineChart</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|react-charting-LineChart.Gaps.chromium.png|1| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>